src/shogun/machine/Pipeline.cpp

```diff
@@ -222,4 +223,37 @@ namespace shogun
 	{
 		return shogun::get<CMachine*>(m_stages.back().second);
 	}
+
+	void CPipeline::set_store_model_features(bool store_model)
+	{
+		get_machine()->set_store_model_features(store_model);
```

Comment:
If Pipeline inherits from CMachine, why cant the method be protected?

Reply:
it's a protected method from base class, you cannot access it of another instance in the derived class `CPipeline`